src/main/java/org/opentripplanner/routing/vehicle_parking/VehicleParkingService.java

```diff
@@ -2,20 +2,57 @@
 
 import java.io.Serializable;
 import java.util.HashSet;
+import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.Stream;
 
+/**
+ * Service that holds all the {@link VehicleParking} instances and an index for fetching parking
+ * locations within a {@link VehicleParkingGroup}.
+ */
 public class VehicleParkingService implements Serializable {
```

Comment:
Is this service thread-safe? If so, add that in the JavaDoc for the class - this is a message for the reader to look inside the class for more documentation...

Reply:
Added javadoc in https://github.com/opentripplanner/OpenTripPlanner/pull/4510/commits/96dbc011165f60776facda314dc7e21f23b6d559 and https://github.com/opentripplanner/OpenTripPlanner/pull/4510/commits/ea5f21a117eda571394820c5ec33542e30bfe0f5